agimus_demo_03_mpc_dummy_traj/launch/bringup_hw.launch.py

```diff
@@ -0,0 +1,164 @@
+    OpaqueFunction,
+    RegisterEventHandler,
+)
+from launch.event_handlers import OnProcessExit
+from launch.launch_description_entity import LaunchDescriptionEntity
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch_ros.substitutions import FindPackageShare
+from launch.substitutions import LaunchConfiguration, PathJoinSubstitution
+from launch_ros.actions import Node
+from controller_manager.launch_utils import generate_load_controller_launch_description
+
+
+def launch_setup(
+    context: LaunchContext, *args, **kwargs
+) -> list[LaunchDescriptionEntity]:
+    robot_ip = LaunchConfiguration("robot_ip")
+    arm_id = LaunchConfiguration("arm_id")
+    load_gripper = LaunchConfiguration("load_gripper")
+    use_fake_hardware = LaunchConfiguration("use_fake_hardware")
+    fake_sensor_commands = LaunchConfiguration("fake_sensor_commands")
+    use_rviz = LaunchConfiguration("use_rviz")
+    package_name = "agimus_demo_02_simple_pd_plus"
+
+    franka_controllers_params = PathJoinSubstitution(
+        [
+            FindPackageShare(package_name),
+            "config",
+            "controller_manager.yaml",
+        ]
+    )
+
+    joint_state_estimator_params = PathJoinSubstitution(
+        [
+            FindPackageShare(package_name),
+            "config",
+            "joint_state_estimator.yaml",
+        ]
+    )
+
+    load_joint_state_estimator = generate_load_controller_launch_description(
+        controller_name="joint_state_estimator",
+        controller_params_file=joint_state_estimator_params,
+    )
+
+    linear_feedback_controller_params = PathJoinSubstitution(
+        [
+            FindPackageShare(package_name),
+            "config",
+            "linear_feedback_controller.yaml",
+        ]
+    )
+
+    load_linear_feedback_controller = generate_load_controller_launch_description(
+        controller_name="linear_feedback_controller",
+        controller_params_file=linear_feedback_controller_params,
+    )
+
+    franka_bringup_launch = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [
+                PathJoinSubstitution(
+                    [
+                        FindPackageShare("agimus_demos_common"),
+                        "launch",
+                        "franka.launch.py",
+                    ]
+                )
+            ]
+        ),
+        launch_arguments={
+            "robot_ip": robot_ip,
+            "arm_id": arm_id,
+            "load_gripper": load_gripper,
+            "use_fake_hardware": use_fake_hardware,
+            "fake_sensor_commands": fake_sensor_commands,
+            "use_rviz": use_rviz,
+            "franka_controllers_params": franka_controllers_params,
+        }.items(),
+    )
+
+    # MPC
+    mpc_controller_yaml = PathJoinSubstitution(
+        [
+            FindPackageShare(package_name),
+            "config",
+            "mpc_controller_params.yaml",
+        ]
+    )
```

Comment:
Maybe it makes more sens to only use the package names?
So instead of MPC: agimus_controller?
```suggestion
    agimus_controller_params = PathJoinSubstitution(
        [
            FindPackageShare(package_name),
            "config",
            "agimus_controller_params.yaml",
        ]
    )
```